src/engine-actions/is-invoice-paid.js

```diff
@@ -13,7 +13,7 @@ const {
  */
 async function isInvoicePaid (paymentRequestHash) {
   const { paymentHash } = await decodePaymentRequest(paymentRequestHash, { client: this.client })
-  const { settled } = await lookupInvoice(paymentHash, { client: this.client })
+  const { settled } = await lookupInvoice({ rHashStr: paymentHash }, { client: this.client })
```

Comment:
What if we did the conversion to bytes in `decodePaymentRequest`? Ideally it would be nice to have a single hash format everywhere outside the lnd-actions directory.

Reply:
I changed it to the conversion in this function. I want to avoid having custom logic in the `lnd-actions` as they are really meant to just be for calling RPCs on LND.